src/crud/useWavesurferRegions.tsx

```diff
@@ -68,10 +68,8 @@ export function useWaveSurferRegions(
     wavesurferRef.current = ws;
     if (ws) {
       ws.on('region-created', function (r: any) {
-        //console.log('region-created', loadingRef.current);
         if (singleRegionRef.current) {
           r.drag = true;
-          if (currentRegion()) currentRegion().remove();
```

Comment:
doesn't this leave multiple regions?

Reply:
This code doesn't work because currentRegion() is always undefined when this method runs. This code runs when the user starts the drag but currentRegion doesn't get set until they finish.